src/arrayMethodJoin.js

```diff
@@ -5,8 +5,18 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
You can delete the `if` statement by added default value to function parametr
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```